reinforcement_learning/rl_mujoco_ray/Dockerfile

```diff
@@ -0,0 +1,94 @@
+ARG AWS_REGION
+ARG CPU_OR_GPU
+ARG SUFFIX
+ARG VERSION
+ARG FRAMEWORK
+
+# use ray 1.6.0 docker
+# TODO: this docker uses tensorflow 2.5.0, use tensorflow 2.7.0 provided by DLC when available
+FROM 462105765813.dkr.ecr.${AWS_REGION}.amazonaws.com/sagemaker-rl-ray-container:ray-1.6.0-${FRAMEWORK}-${CPU_OR_GPU}-${SUFFIX}
```

Comment:
Is this account ID the same across all regions?

Reply:
Yes it is.